packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -299,7 +299,10 @@ const getMinAndMaxReferenceLines = (
             if (!serie.markLine) return [];
             return serie.markLine?.data.reduce<number[]>((acc, data) => {
                 try {
-                    const value = parseInt(data[axis]);
+                    const axisValue =
+                        axis === 'yAxis' ? data.yAxis : data.xAxis;
+                    if (axisValue === undefined) return acc;
```

Comment:
Why would it be undefined?

Reply:
because both yAxis and xAxis can be undefined.